gluoncv/utils/viz/bbox.py

```diff
@@ -102,3 +102,100 @@ def plot_bbox(img, bboxes, scores=None, labels=None, thresh=0.5,
                     bbox=dict(facecolor=colors[cls_id], alpha=0.5),
                     fontsize=12, color='white')
     return ax
+
+def cv_plot_bbox(img, bboxes, scores=None, labels=None, thresh=0.5,
+                 class_names=None, colors=None,
+                 absolute_coordinates=True, scale=1.0):
+    """Visualize bounding boxes with OpenCV.
+
+    Parameters
+    ----------
+    img : numpy.ndarray or mxnet.nd.NDArray
+        Image with shape `H, W, 3`.
+    bboxes : numpy.ndarray or mxnet.nd.NDArray
+        Bounding boxes with shape `N, 4`. Where `N` is the number of boxes.
+    scores : numpy.ndarray or mxnet.nd.NDArray, optional
+        Confidence scores of the provided `bboxes` with shape `N`.
+    labels : numpy.ndarray or mxnet.nd.NDArray, optional
+        Class labels of the provided `bboxes` with shape `N`.
+    thresh : float, optional, default 0.5
+        Display threshold if `scores` is provided. Scores with less than `thresh`
+        will be ignored in display, this is visually more elegant if you have
+        a large number of bounding boxes with very small scores.
+    class_names : list of str, optional
+        Description of parameter `class_names`.
+    colors : dict, optional
+        You can provide desired colors as {0: (255, 0, 0), 1:(0, 255, 0), ...}, otherwise
+        random colors will be substituted.
+    absolute_coordinates : bool
+        If `True`, absolute coordinates will be considered, otherwise coordinates
+        are interpreted as in range(0, 1).
+    scale : float
+        The scale of output image, which may affect the positions of boxes
+
+    Returns
+    -------
+    numpy.ndarray
+        The image with detected results.
+
+    """
+    from matplotlib import pyplot as plt
+    from ..filesystem import try_import_cv2
+    cv2 = try_import_cv2()
+
+    if labels is not None and not len(bboxes) == len(labels):
+        raise ValueError('The length of labels and bboxes mismatch, {} vs {}'
+                         .format(len(labels), len(bboxes)))
+    if scores is not None and not len(bboxes) == len(scores):
+        raise ValueError('The length of scores and bboxes mismatch, {} vs {}'
+                         .format(len(scores), len(bboxes)))
+
+    if len(bboxes) < 1:
+        return img
+
+    if isinstance(bboxes, mx.nd.NDArray):
+        bboxes = bboxes.asnumpy()
+    if isinstance(labels, mx.nd.NDArray):
+        labels = labels.asnumpy()
+    if isinstance(scores, mx.nd.NDArray):
+        scores = scores.asnumpy()
+
+    if not absolute_coordinates:
+        # convert to absolute coordinates using image shape
+        height = img.shape[0]
+        width = img.shape[1]
+        bboxes[:, (0, 2)] *= width
+        bboxes[:, (1, 3)] *= height
+    else:
+        bboxes *= scale
```

Comment:
only `True` if `absolute_coordinates=True`

Reply:
I couldn't get your comment here. Would you please explain more?